src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -5923,6 +5923,7 @@ def test_network_service_aliases(self, resource_group):
 
 class NetworkBastionHostScenarioTest(ScenarioTest):
 
+    @unittest.skip('az network bastion update: InvalidRequestContent The request content was invalid and could not be deserialized: Required property `name` not found in `sku`.' )
```

Comment:
@necusjz At present, this test cannot be passed in live mode. To avoid blocking this PR merge, we skip this test first. Could you help to submit a separate new PR to fix the live test?

Reply:
FYI. Azure Bastion will be moved to extension as planned.

Reply:
Got it, thanks~